modules/python/init.zsh

```diff
@@ -125,7 +125,7 @@ if (( $+VIRTUALENVWRAPPER_VIRTUALENV || $+commands[virtualenv] )) \
   else
     # Fallback to 'virtualenvwrapper' without 'pyenv' wrapper if 'python' is
     # available in '$path'.
-    if (( ! $+VIRTUALENVWRAPPER_PYTHON )) && (( $#commands[(i)python[23]#] )); then
+    if (( ! $+VIRTUALENVWRAPPER_PYTHON )) && (( $+commands[(i)python[23]] )); then
```

Comment:
This line appeared to still work for me, but given that the syntax above was broken, I'd feel more comfortable reverting this one as well, at least until we understand what exactly it's trying to get at.